### PR TITLE
Add comments to tests

### PR DESCRIPTION
**Pull request by ryanjduffy**

### CODE REVIEW

**Review by jasonLaster — APPROVED** [review]
Wow! really simple

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -130,3 +130,24 @@ jobs:
...
+    needs: [e2etest, generate-test-run-id]
+    runs-on: ubuntu-latest
+    if: ${{ always() }}
+    steps:
+    - uses: actions/download-artifact@v3
+    - uses: actions/github-script@v6
+      id: combine-recordings
+      with:
+        script: |
+          const shards = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"];
```
Comment (markerikson) [inline .github/workflows/test.yml:144]: :speech_balloon: wait, you can _do_ this? 🤯

**Comment by jasonLaster** [comment]
****. [View test run↗︎](https://vercel.link/github-learn-more)


<details open=true>
  <summary><b>2 Failed</b></summary>
<ul>
<li><a href="https://github.com/notifications?query=org%3Areplayio">cypress/e2e/session.cy.ts</a> 
<li><a href="https://github.com/notifications?query=org%3Areplayio">cypress/e2e/subscriptions.cy.ts</a> 
<li><a href="https://github.com/notifications?query=org%3Areplayio">cypress/e2e/navigation.cy.ts</a> 
</ul>
</details>


<details>
  <summary><b>3 Passed</b></summary>
<ul>
<li><a href="https://github.com/notifications?query=org%3Areplayio">cypress/e2e/session.cy.ts</a> 
<li><a href="https://github.com/notifications?query=org%3Areplayio">cypress/e2e/subscriptions.cy.ts</a> 
<li><a href="https://github.com/notifications?query=org%3Areplayio">cypress/e2e/navigation.cy.ts</a> 
</ul>
</details>

**Comment by ryanjduffy** [comment]
Closed in favor of #7352

**Pull request closed without merging**